cobalt/build/test_targets.py

```diff
@@ -37,6 +37,7 @@
     '//net:net_unittests',
     '//skia:skia_unittests',
     '//starboard/nplb:nplb',
+    '//third_party/blink/renderer/controller:blink_unittests',
```

Comment:
As discussed offline, this has to go in the appropriate platform json in https://github.com/youtube/cobalt/tree/main/cobalt/build/testing/targets

At least until test targets are calculated automatically.